src/PresentationalComponents/Dashboard/_dashboard.scss

```diff
@@ -111,6 +111,7 @@
       }
 
       > * > .pf-c-card {
+        .pf-c-card__body { z-index: 2; }
```

Comment:
right now this sets the z-index of all card bodies to a z-index of 2 when the screen is over a max-width of 1200px. is this the intention? or should it persist on all screen sizes? also setting this on all card bodies could be problematic if someone adds say a dropdown in the card, the menu has a z-index set on it also. is it possible to scope the styles directly to the tooltip, or have a new class for just this card body?